pages/about.html

```diff
@@ -4,13 +4,32 @@
 type: default
 ---
 
-<h1>About</h1>
-<p class="page-description">CEFACT Linked Data v1 vocabulary</p>
-<p>Machine Readable Term Definitions:</p>
-<p>JSON-LD vocabularies are accessible using HTTP content negotiation.</p>
-<code>curl https://vocabulary.uncefact.org/ -H 'Accept: application/ld+json'</code>
-<code>curl https://vocabulary.uncefact.org/unlocode# -H 'Accept: application/ld+json'</code>
-<p/>
-<p>JSON-LD Context files are accessible using the direct links.</p>
+<h1>UN/CEFACT Web Vocabulary</h1>
+<p class="page-description">https://vocabulary.uncefact.org/</p>
+
+<h2>About</h2>
+<p>
+    The UN/CEFACT Web Vocabulary is a linked data representation of the UN/CEFACT
+    <a href="https://unece.org/fileadmin/DAM/cefact/brs/BuyShipPay_BRS_v1.0.pdf">Buy-Ship-Pay Reference Data Model</a>.
+    The BSP RDM generalizes concepts of Multi-Modal Transport Reference Data Model (MMT-RDM) and the Supply Chain
+    Reference Data Model (SCRDM), which can be applied by country and regional administrations and industries.
+</p>
+
+<h2>Version</h2>
+<p>The vocabulary reflects BSP version 20A.</p>
```

Comment:
```suggestion
<p>The vocabulary reflects BSP version Draft 20A.</p>
```
as per https://github.com/uncefact/spec-jsonld/issues/147#issuecomment-1327113270

Reply:
The inout file was called 20A.